test/api/delegates.js

```diff
@@ -629,6 +629,31 @@ describe('GET /api/delegates', function () {
 			done();
 		});
 	});
+
+	it('using orderBy with random sort field should not place NULLs first', function (done) {
+		var delegatesSortFields = ['approval', 'productivity', 'rate', 'vote'];
+		var randomSortField = delegatesSortFields[Math.floor(Math.random() * delegatesSortFields.length)];
```

Comment:
any of these fields shouldn't be nulls, but they can be referring to fields sql definitions. No idea which one to choose. @karmacoma what do you think?